polaris-react/src/components/ActionMenu/components/SecondaryAction/SecondaryAction.tsx

```diff
@@ -30,9 +32,11 @@ export function SecondaryAction({
   }, [getOffsetWidth]);
 
   const buttonMarkup = (
-    <Button onClick={onAction} {...rest}>
-      {children}
-    </Button>
+    <span ref={activatorRef}>
+      <Button onClick={onAction} {...rest}>
+        {children}
+      </Button>
+    </span>
```

Comment:
Allows to return focus to the action within Secondary actions

https://user-images.githubusercontent.com/104942025/226448407-69383eed-d9f3-4e88-9041-d021c6472ced.mov